vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -84,6 +84,9 @@ type QMPConfig struct {
 	// logger is used by the qmpStart function and all the go routines
 	// it spawns to log information.
 	Logger QMPLog
+
+	// specify the capacity of buffer used by receive QMP response.
```

Comment:
All files below `vendor/*` are "pristine" / unmodified copies of upstream package versions. Rather than changing this file, you need to:

- Raise a PR on `govmm`.
- Once that PR is merged, raise a PR on this repo that includes a re-vendor commit to "copy in" the new version of this file into `vendor/*`.

See the following for further details:

- https://github.com/kata-containers/community/blob/master/VENDORING.md
- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#re-vendor-prs

Reply:
I have create a new PR intel/govmm#104, I will update commit after it mergerd.

Reply:
Thanks - I see that is now merged.